### PR TITLE
Align error_dashboard with the demo

### PR DESCRIPTION
**Pull request by tdhellmann**
Removed a redundant panel (since @pupadhyaya14 's last merge) and added another sourcetype to the internal errors.

### CODE REVIEW

File: packages/guidance-app-cloud/src/main/resources/splunk/default/data/ui/views/error_dashboard.xml [hunk 1]
```diff
@@ -108,7 +95,7 @@
       <event>
         <title>Splunk internal Errors</title>
         <search>
-          <query>index=_* AND sourcetype IN ("python", "splunkd", "web") AND "$appid$" AND log_level IN ($log_level$)</query>
+          <query>index=_* AND sourcetype IN ("python", "splunkd", "web", "splunk_search_messages") AND "$appid$" AND log_level IN ($log_level$)</query>
```
Comment (pupadhyaya14) [inline packages/guidance-app-cloud/src/main/resources/splunk/default/data/ui/views/error_dashboard.xml:98]: I believe we don't need `AND "$appid$"` here as well.

**Review by pupadhyaya14 — APPROVED** [review]
Just a small comment

File: packages/guidance-app-cloud/src/main/resources/splunk/default/data/ui/views/error_dashboard.xml (hunk 1 above)
Comment (tdhellmann) [reply packages/guidance-app-cloud/src/main/resources/splunk/default/data/ui/views/error_dashboard.xml:98]: For the "Splunk Internal Errors" dashboard we do (I think), but I'll remove it from "User App Errors"

**Comment by tdhellmann** [comment]
Trying to get this branch back into sync

**Pull request closed without merging**